src/api/mgmt.js

```diff
@@ -11,20 +11,28 @@ const BaseApi = require("../utils/baseApi");
 const MGMT_API_ENDPOINT_PROD = "https://managementapi.emp.ebsd.ericsson.net";
 const MGMT_API_ENDPOINT_STAGE =
   "https://psempempmanagementapi.ebsd.ericsson.net";
-const apiKeyId = process.env.API_KEY_ID;
-const apiKeySecret = process.env.API_KEY_SECRET;
+const env_apiKeyId = process.env.API_KEY_ID;
+const env_apiKeySecret = process.env.API_KEY_SECRET;
 
 class EnigmaManagementAPI extends BaseApi {
   constructor(customerUnit, businessUnit, options) {
-    debug("API_KEY_ID=%s, API_KEY_SECRET=*****", apiKeyId);
+    debug("API_KEY_ID=%s, API_KEY_SECRET=*****", env_apiKeyId);
     options = {
       environment: "production",
       ...options,
     };
+
     const url =
       options.environment === "stage"
         ? MGMT_API_ENDPOINT_STAGE
         : MGMT_API_ENDPOINT_PROD;
+
+    const apiKeyId = options.API_KEY_ID ? options.API_KEY_ID : env_apiKeyId;
```

Comment:
You don't need the variables in the top of the file as far as I can tell? . Just do const apiKey = options.API_KEY_ID || process.env.API_KEY_ID, and same approach for API_KEY_SECRET

Reply:
Indeed. I went with the path of least resistens here and left the declaration were it was because i didn't know were else it was used. But initial testing seem to pan out so went with your change